types/fs-extra/fs-extra-tests.ts

```diff
@@ -142,11 +145,11 @@ fs.writeJSON(file, object, writeOptions).then(() => {
 fs.writeJsonSync(file, object, writeOptions);
 fs.writeJSONSync(file, object, writeOptions);
 
-fs.ensureDir(path).then(() => {
+fs.ensureDir(path, ensureOptions).then(() => {
```

Comment:
These tests should be additive, as I believe the previous examples are still valid.

Reply:
I agree, let me change it